Runtime/Scripts/msCoreAPI.cs

```diff
@@ -24,10 +24,15 @@ internal static class Lib {
 
     #endregion
 
-
-    internal static string GetPluginVersion() {
-        IntPtr nativeStr = msGetPluginVersionStr(); //Not direct marshalling because there is no free on C# side.
-        return Marshal.PtrToStringAnsi(nativeStr);
+    static string version;
+    internal static string GetPluginVersion()
+    {
+        if (version == null)
```

Comment:
What is the optimization? was this invoked in a loop?

Reply:
Yeah in a few places and every draw call when it draws the library version.